### PR TITLE
Automation/SDK fixes for MLB Deprecation

### PR DESCRIPTION
**Pull request by ramrutha497**
_This is the default pull request template. You can customize it by adding a `pull_request_template.md` at the root of your repo or inside the `.github` folder._

# Jira Ticket
https://spotinst.atlassian.net/browse/SPOTAUT-17868

_Include a link to your Jira Ticket_
_Example: [JIRAISS-1234](https://spotinst.atlassian.net/browse/JIRAISS-1234)_

# Demo

_Please add a recording of the feature/bug fix in work. if you added new routes, the recording should show the request and response for each new/changed route_

# Checklist:
- [ ] I have filled relevant self assessment ([NodeJS](https://docs.google.com/forms/d/e/1FAIpQLSfl14u9AOBAmxVJ272tvO7XNuXE-EMvWaGcaRZalu1UAKB7RA/viewform), [Frontend](https://docs.google.com/forms/d/e/1FAIpQLSdiBPNKH81w_EkavihVL8Uwb0j7tP8PwJmLFYm2nCOQxz-1qw/viewform), [Backend](https://docs.google.com/forms/d/e/1FAIpQLSed_PsTJ5-XIWkFL6BSDE2AQRBVPmwc3PAmHZkUn-erzVI37Q/viewform?usp=sf_link))
- [ ] I have run ESlint on my changes and fixed all warnings and errors (**NodeJS & Frontend Services**)
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have validated all the requirements in the Jira task were answered
- [ ] I have all neccessary approvals for the design/mini design of this task
- [ ] I have approved the API changes and granular permission patterns (documentation subtask) (**For public services only**)

### CODE REVIEW

File: spotinst_sdk2/test/managed_instance/aws/test_aws_ma_model.py [hunk 1]
```diff
@@ -79,8 +79,7 @@ def runTest(self):
 
 class TestAwsManagedInstanceIntegrationsLoadBalancersConfiguration(AwsManagedInstanceTestCase):
     def runTest(self):
-        lb = LoadBalancer(name="name", arn="arn", type="MULTAI_TARGET_SET", balancer_id="lb-1ee2e3q",
-                          target_set_id="ts-3eq", az_awareness=True, auto_weight=True)
+        lb = LoadBalancer(name="name", arn="arn")
```
Comment (chandra1-n) [inline spotinst_sdk2/test/managed_instance/aws/test_aws_ma_model.py]: type is required either CLASSIC or TARGET_GROUP

File: spotinst_sdk2/test/test_lib/input/elastigroup/aws_group.json [hunk 2]
```diff
@@ -187,8 +187,7 @@
               "targetSetId": "ts-123456789",
               "balancerId": "lb-123456789",
               "autoWeight": true,
-              "azAwareness": false,
-              "type": "MULTAI_TARGET_SET"
+              "azAwareness": false
```
Comment (chandra1-n) [inline spotinst_sdk2/test/test_lib/input/elastigroup/aws_group.json]: this block is not required as  it is related to multai_target_sets

File: spotinst_sdk2/test/test_lib/input/managed_instance/aws_managed_instance.json [hunk 3]
```diff
@@ -111,12 +111,7 @@
         "loadBalancers": [
           {
             "name": "name",
-            "arn": "arn",
-            "type": "MULTAI_TARGET_SET",
-            "balancerId": "lb-1ee2e3q",
-            "targetSetId": "ts-3eq",
-            "azAwareness": true,
-            "autoWeight": true
+            "arn": "arn"
```
Comment (chandra1-n) [inline spotinst_sdk2/test/test_lib/input/managed_instance/aws_managed_instance.json]: type is required here, either "CLASSIC" or "TARGET_GROUP"

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix the review comments

File: spotinst_sdk2/test/managed_instance/aws/test_aws_ma_model.py (hunk 1 above)
Comment (anuragsharma-123) [reply spotinst_sdk2/test/managed_instance/aws/test_aws_ma_model.py]: Fixed.

File: spotinst_sdk2/test/test_lib/input/managed_instance/aws_managed_instance.json (hunk 3 above)
Comment (anuragsharma-123) [reply spotinst_sdk2/test/test_lib/input/managed_instance/aws_managed_instance.json]: Fixed.

File: spotinst_sdk2/test/test_lib/input/elastigroup/aws_group.json (hunk 2 above)
Comment (anuragsharma-123) [reply spotinst_sdk2/test/test_lib/input/elastigroup/aws_group.json]: Fixed.

**Review by chandra1-n — APPROVED** [review]

**Pull request merged**